shared/simplestreams/products.go

```diff
@@ -30,6 +30,7 @@ type Product struct {
 	ReleaseTitle    string                    `json:"release_title"`
 	Supported       bool                      `json:"supported,omitempty"`
 	SupportedEOL    string                    `json:"support_eol,omitempty"`
+	Variant         string                    `json:"variant,omitempty"`
```

Comment:
@stgraber worth adding a comment to the fields that diverge from the 'standard' simplestreams format?